src/containers/board-container.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Board } from '../components/board.js';
+import {
+    getNewBoard,
+    flipTheLights,
+    getLights,
+    checkForWin,
+} from '../game-logic.js';
+
+export class BoardContainer extends React.Component {
+    state = {
+        numMoves: 0,
+        hasWon: false,
+    };
+
+    board = this.props.startingBoard ? this.props.startingBoard : getNewBoard();
```

Comment:
Instead of using a conditional here, I'd probably go up to pages/index.js and pass `startingBoard={getNewBoard()}` to it. That way, the BoardContainer doesn't have any test-specific logic, it's always using the same logic to setting up a new board. (Not a huge deal, though)